ggshield/cmd/iac/scan/scan.py

```diff
@@ -87,9 +56,20 @@ def scan_cmd(
     if directory is None:
         directory = Path().resolve()
     update_context(ctx, exit_zero, minimum_severity, ignore_policies, ignore_paths)
-    result = iac_scan(ctx, directory)
-    scan = IaCScanCollection(id=str(directory), type="path_scan", result=result)
+    result = (
+        iac_scan(ctx, directory)
+        if since is None
+        else iac_diff_scan(ctx, directory, since, staged)
```

Comment:
If I read correctly, the command is not correct, maybe that it on purpose and will be in another MR ?